### ISSUE TITLE
Custom resource deployment fails in v6.0.0

### ISSUE THREAD

**Issue by bchunduri** [aws-solutions/qnabot-on-aws#749]
Hi Team,

We are deploying the new version of AWS QnABot v6.0.0 using the Cloudformation stacks and it failed to create all the custom resources with error statement as shown in below example:

**CloudFormation did not receive a response from your Custom Resource. Please check your logs for requestId [xxxxxxx-xxxxxx-xxxxxx-xxxx]. If you are using the Python cfn-response module, you may need to update your Lambda function code so that CloudFormation can attach the updated version.**

There were no cloudwatch events recorded against any of the requestId's from the failed resources and the triggered lambda logs throwing this error: 

"errorType": "Runtime.ImportModuleError",
"errorMessage": "Error: Cannot find module 'index'\nRequire stack:\n- /var/runtime/index.mjs",

Please investigate, review and assist us to unblock this problem. Appreciate your swift action.

**Comment by abhirpat** [comment]
Hi @bchunduri, thank you for reporting this issue. 

Please try following:

-  Could you please try clicking on stack that has failed > Events > Detect RootCause and see if it points you to the actual error? Also, please try selecting all lambdas and run cloudwatch query mentioned in the section [Use Log Insights to query logs from CloudWatch groups](https://docs.aws.amazon.com/solutions/latest/qnabot-on-aws/tuning-testing-and-troubleshooting-unexpected-answers.html).
- Is your stack name longer than 26 character ? If so, please try using using latest version which has the fix (see [CHANGELOG](https://github.com/aws-solutions/qnabot-on-aws/blob/main/CHANGELOG.md)).

Additionally:
1. Are you using main or VPC template or extended template (deployed using Github repo and npm commands)? 
2. What are your configurations in Cloudformation ? Could you please provide steps to reproduce ?

Thanks,
Abhishek

**Comment by bchunduri** [comment]
Hi @abhirpat, thanks for the quick suggestions and let me review your points:

1. Unfortunately log insights query could not display any events of the failed resource. And by debugging and backtracking, it is assume to throw errors onto the cfn-proxy log group.
2. The root stack is exactly 26 characters and other nested stacks go on a higher count and if that is the case, I need to either adapt that little change fixing this bug in v6.0.1
3. I am using the vpc template and passed all relevant custom parameters to the main and all nested stacks created by the root stack


Thanks,

Babu

**Comment by abhirpat** [comment]
@bchunduri Thank you. Could you please try using latest template v6.0.2 and update us know if you have this issue?

**Comment by michaelin-96** [comment]
@bchunduri, I deployed v6.0.0 VPC template with 26 characters in the stack name and wasn't able to replicate the issue. I was able to deploy successfully.

**Comment by bchunduri** [comment]
Hi @michaelin-96 , thanks and have you tried stack names with more than 26 characters ?

**Comment by bchunduri** [comment]
@abhirpat , wonder to see another minor version released in no time ! This leave us in dilemma to hold over to a stable release :(

**Comment by DeccanAjay** [comment]
@bchunduri - thank you for bringing this to our attention. Please use the latest version to update your stack. We will look into this more and be in touch shortly.

**Comment by fhoueto-amz** [comment]
Hi @bchunduri 
I am closing this. Please feel free to reopen or open a new issue, if you have more details to help reproduce.
